RootMap not installed in right place

The RootMap is not installed in right lib folder

Just changing that line gives
```
CMake Error at cmake/DD4hep.cmake:112 (install):
  install FILES given no DESTINATION!
```
if the variable is not set in the cmake call.

--- a/cmake/DD4hep.cmake
+++ b/cmake/DD4hep.cmake
@@ -109,8 +109,13 @@ function(dd4hep_generate_rootmap_notapple library)
   #                   DEPENDS ${library})
   ##add_custom_target(${library}Rootmap ALL DEPENDS ${rootmapfile})
 
+  SET( install_destination "lib" )
+  if( CMAKE_INSTALL_LIBDIR )
+    SET( install_destination ${CMAKE_INSTALL_LIBDIR} )
+  endif()
+
   install(FILES ${LIBRARY_OUTPUT_PATH}/${rootmapfile}
-    DESTINATION lib
+    DESTINATION ${install_destination}
   )
 endfunction()
 #

